WalletWasabi.Fluent/ViewModels/Login/PasswordFinder/SearchPasswordViewModel.cs

```diff
@@ -65,6 +65,7 @@ private void FindPassword(PasswordFinderOptions options, CancellationToken token
 			}
 			catch (OperationCanceledException)
 			{
+				Logger.LogTrace("Operation was canceled.");
```

Comment:
> Error CS0103: The name 'Logger' does not exist in the current context

You need to add `using WalletWasabi.Logging;`

Reply:
Thanks. Added in https://github.com/zkSNACKs/WalletWasabi/pull/6648/commits/c7ad4f9fd550fbc046c686f8524d330dd8fa3ee6